crates/turbopack-dev-server/src/source/router.rs

```diff
@@ -9,28 +12,33 @@ use crate::source::ContentSourcesVc;
 /// ContentSource will serve all other subpaths.
 #[turbo_tasks::value(shared)]
 pub struct RouterContentSource {
+    pub base_path: OptionStringVc,
     pub routes: Vec<(String, ContentSourceVc)>,
     pub fallback: ContentSourceVc,
 }
 
 impl RouterContentSource {
-    fn get_source<'s, 'a>(&'s self, path: &'a str) -> (&'s ContentSourceVc, &'a str) {
+    async fn get_source<'s, 'a>(&'s self, path: &'a str) -> Result<(&'s ContentSourceVc, &'a str)> {
```

Comment:
Nit: Why was this made async?

Reply:
Previous version used `base_path`, no longer needed.